python/cac_tripplanner/destinations/management/commands/load_events.py

```diff
@@ -90,4 +91,4 @@ def handle_feed_item(self, item):
 
         feed_event, created = FeedEvent.objects.update_or_create(guid=guid, defaults=updated_item)
         if created:
-            self.stdout.write('Added event with guid: "%s"' % guid)
+            self.stdout.write('%s: Added event with guid: "%s"' % (str(datetime.utcnow()), guid))
```

Comment:
I think I missed this before, but we should use the new-style string interpolation for these.